desktop_version/src/Game.cpp

```diff
@@ -2905,32 +2943,33 @@ void Game::updatestate(void)
             break;
         }
         case 4003:
-            state = 0;
+            state++;
```

Comment:
I don't think this ends up causing any issues, but why was this changed to `state++;`? It should be left the same.